CHANGELOG.md

```diff
@@ -1,16 +1,4 @@
-## 0.5.0-beta+1
-
-This code is considered production quality, but depends on angular2:
-^3.0.0-alpha+1. The alpha tag represents the evolving nature of the AngularDart
-api, not code quality (3.0.0-beta+1 is used in production Google apps).
-
- * Fix link in CHANGELOG to old pub package.
-
-## 0.5.0-beta
-
-This code is considered production quality, but depends on angular2:
-^3.0.0-alpha+1. The alpha tag represents the evolving nature of the AngularDart
-api, not code quality (3.0.0-beta+1 is used in production Google apps).
+## 0.5.0
 
```

Comment:
Do you really want to remove these lines of the change log? Seems weird to have versions published to pub that don't have any description.